### ISSUE TITLE
Schema is not defined

### ISSUE THREAD

**Issue by ZahidTekbas** [isar/isar#228]
**Steps to Reproduce**
I have decided to use [Isar database](https://pub.dev/packages/isar) in my next project and I find it much helpful when dealing with local data.

I followed the [quickstart guide](https://isar.dev/tutorials/quickstart.html) in its website. I added dependencies. Annotated the contact class. Ran code generator. But at fourth step, I have problem creating schema while creating Isar instance.

**Code sample**
```
initIsar() async {
  final dir = await getApplicationSupportDirectory();
  final isar = await Isar.open(
    schemas: [ContactSchema],
    directory: dir.path,
    inspector: true,
  );
}


```
The problem is where I typed ContactSchema, it says
```

Undefined name 'ContactSchema'.
Try correcting the name to one that is defined, or defining the name.
```

So question I wan to ask is, I followed guide but I'm unable to create a schema. How can I create one to make Isar db work?

**Version**
 - Platform: iOS Emulator
 - Flutter 2.8.1 • channel stable
 - Isar version: ^2.1.4
 - Isar Generator version: ^2.1.4

**Comment by ZahidTekbas** [comment]
Yes I did it. This is the example class I copied from the guide: 

```
import 'package:isar/isar.dart';

@Collection()
class Contact {
  @Id()
  int? id;

  late String name;
}
```

**Comment by simonpham** [comment]
You need to add `part 'contact.g.dart';` then run `flutter pub build_runner build` again.

```
import 'package:isar/isar.dart';

part 'contact.g.dart';

@Collection()
class Contact {
  @Id()
  int? id;

  late String name;
}
```
